terasoluna-gfw-common/src/main/java/org/terasoluna/gfw/common/codelist/validator/ExistInCodeListValidatorForCharSequence.java

```diff
@@ -26,16 +26,16 @@
  * </p>
  */
 
-public class ExistInCodeListValidatorForString
+public class ExistInCodeListValidatorForCharSequence
                                               extends
-                                              AbstractExistInCodeListValidator<String> {
+                                              AbstractExistInCodeListValidator<CharSequence> {
 
     /**
      * Fetches the code value which is the target of validation
      * @see org.terasoluna.gfw.common.codelist.validator.AbstractExistInCodeListValidator#getCode(Object)
      */
     @Override
-    protected String getCode(String value) {
-        return value;
+    protected String getCode(CharSequence value) {
+        return value == null ? null : value.toString();

```

Comment:
- [ ] Why is ``toString()`` required? In my understanding, it is possible to return ``value`` as it is because ``String`` implements ``CharSequence``.